ca_description/urdf/create_base.xacro

```diff
@@ -54,6 +55,13 @@
     <xacro:create_wheel prefix="right" y_offset="${wheel_separation / -2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:caster_wheel/>
 
+    <xacro:light_sensor prefix="left">
```

Comment:
Call the parameter `prefix_name` or whatever you consider but it was not clear to me the purpose of that parameter.